recipe/meta.yaml

```diff
@@ -13,16 +13,16 @@ source:
   '{{ hash_type }}': '{{ hash_value }}'
 
 build:
-  number: 1
+  number: 2
   entry_points:
     - get_juldate = pydl.goddard.astro:get_juldate_main
     - hogg_iau_name = pydl.pydlutils.misc:hogg_iau_name_main
     - compute_templates = pydl.pydlspec2d.spec1d:template_input_main
-  script: python setup.py install --offline --single-version-externally-managed --record=record.txt
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
```

Comment:
```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Reply:
That made no difference.

Reply:
Ah, see https://github.com/conda-forge/conda-smithy/issues/1182